Dockerfile.release

```diff
@@ -0,0 +1,5 @@
+FROM debian:11-slim
```

Comment:
Any reason to use `debian:11-slim` specifically here? Should be fine, I'm just not aware of anywhere else we use it.

Reply:
That's the image layer `avalanche-cli` uses. I was running into some compatability issues with the golang bullseye layer, so I elected to just go with the battle tested option.